pulp_python/tests/functional/api/test_crud_content_unit.py

```diff
@@ -1,18 +1,26 @@
+# coding=utf-8
+"""Tests that perform actions over content unit."""
 import unittest
 
-from requests.exceptions import HTTPError
-
-from pulp_smash import api, config, utils, exceptions
 from pulp_smash.pulp3.utils import delete_orphans
 
-from pulp_python.tests.functional.constants import (
-    PYTHON_CONTENT_PATH,
-    PYTHON_REPO_PATH,
-    PYTHON_WHEEL_URL,
-    PYTHON_WHEEL_FILENAME
+from pulp_python.tests.functional.utils import (
+    gen_artifact,
+    gen_python_client,
+    gen_python_content_attrs,
+    monitor_task,
+    skip_if,
 )
-from pulp_python.tests.functional.utils import skip_if
 from pulp_python.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
+from tempfile import NamedTemporaryFile
+
+from pulpcore.client.pulp_python import RepositoriesPythonApi, ContentPackagesApi
+from pulp_smash.utils import http_get
+from pulp_python.tests.functional.constants import (
+    PYTHON_PACKAGE_DATA,
+    PYTHON_EGG_FILENAME,
+    PYTHON_EGG_URL,
+)
 
 
 class OneShotUploadTestCase(unittest.TestCase):
```

Comment:
I think the tests in this class should either be moved to ```ContentUnitTestCase``` unless there's a good reason to keep them separate.  They are combined in the RPM tests.